src/config/firebase.ts

```diff
@@ -0,0 +1,10 @@
+export const firebaseConfig = {
+  apiKey: "AIzaSyBnuxL3OFy6EbR5T7XlkehkyXbOxnswQtQ",
+  authDomain: "mutualaid-757f6.firebaseapp.com",
+  databaseURL: "https://mutualaid-757f6.firebaseio.com",
+  projectId: "mutualaid-757f6",
+  storageBucket: "mutualaid-757f6.appspot.com",
+  messagingSenderId: "57281805127",
+  appId: "1:57281805127:web:7c025105ef5fb5e5232a62",
+  measurementId: "G-7L9JFWQHS2"
+};
```

Comment:
Can we set these as environment variables instead of hardcoding it?

Reply:
These aren't private keys, they're just used to communicate with GCP. Not sure how having to run `npm start` with environment variables would be an improvement. In the name of speed this doesn't feel like a priority

Reply:
Agree, we can wait with environment variables